docs/self-managed/identity/orchestration-identity/installation.md

```diff
@@ -0,0 +1,53 @@
+---
+id: installation
+title: Installation
```

Comment:
I am not sure if "Installation" is the best name for this page anymore. "Installation and setup"? "Initial setup"?

Reply:
I think out of the options @conceptualshark I'd opt for `Initial setup`, as its no longer just about installation but instead about the initial points of installation and configuration